### ISSUE TITLE
collector_id needs to be unique for each server?

### ISSUE THREAD

**Issue by msteelepmp** [Graylog2/documentation#312]
It appears the collector_id needs to be unique for each server, or the http://mygraylogserver/system/collectors won't show the 2 servers separately (as one would expect).

## Context

I'd recommend adding a note to the table saying that collector_id needs to be unique for each server.

## Expected Content

collector_id - should be unique for each collector_sidecar instance (aka server sending logs to graylog)

## Your Environment

graylog client:
  Amazon Linux AMI 2017.03
  collector-sidecar-0.1.1-1.x86_64.rpm

* Graylog Version: 2.2.3 Omnibus
* Elasticsearch Version:
* MongoDB Version:
* Operating System: Ubuntu 14.04 (graylog server)

**Comment by joschi** [comment]
@msteelepmp It already says "Unique ID (UUID) of the instance.".

How would you rephrase it to be more clear?

Hint: You can [create a pull request](https://help.github.com/articles/about-pull-requests/) with your suggestions.

**Comment by dennisoelkers** [comment]
We are using GitHub issues for tracking bugs in Graylog itself, but this doesn't look like one. Please post this issue to our [public mailing list](https://groups.google.com/forum/#!forum/graylog2) or join the [#graylog channel on freenode IRC](https://webchat.freenode.net/?channels=%23graylog).

Thank you!
